### PR TITLE
1.12/1645 remove need for hand inputting numbers and just scan only fraud response

### PR DESCRIPTION
**Pull request by AlinChindea**
### Trello :ticket: 🔗 [https://trello.com/c/2Hq6Y6cc/1645-remove-need-for-hand-inputting-numbers-and-just-scan-only-fraud-response-vvv-eee](https://trello.com/c/2Hq6Y6cc/1645-remove-need-for-hand-inputting-numbers-and-just-scan-only-fraud-response-vvv-eee)

Linked to this:  neontribe/ARCVMarket#222

### CODE REVIEW

File: database/seeds/UsersSeeder.php [hunk 1]
```diff
@@ -60,5 +60,8 @@ public function run()
 
         // So we reliably have one user who can act on behalf of all traders.
         $users[3]->traders()->sync($traders);
+
+        // So we reliably have a specific user who cannot access the tap page
+        $users[1]->traders()->sync($traders[6]);
```
Comment (AlinChindea) [inline database/seeds/UsersSeeder.php:65]: @charlesstrange2 this is not fully clear to me - I wanted my specific trader to be assigned to a specific user.

Comment (charlesstrange2) [reply database/seeds/UsersSeeder.php:65]: the relationship of Users to Traders is Many to Many
so $users[1] gets a user off the array
... you then pull the users' traders() relationship 
... and sync() an array of traders ids to it.
... in this case 1 of them 

sync manages the pivot table (trader_users in the DB) that sits between the "user M>--<M trader" relationship.

File: database/seeds/SponsorsSeeder.php [hunk 2]
```diff
@@ -42,6 +42,10 @@ public function run()
 
         // 4MAY20-VC1-CH1P-HA-012020 is in sponsor 5, unmodified
         // 1MAY20a-VC2-CH1-HI-042019 is in sponsor 5, unmodified
+
+        $noTapSponsor = factory(App\Sponsor::class)->create(['name' => "No Tap Project", "can_tap" => false]);
+        $noTapSponsor->evaluations()->saveMany($this->qualifyPrimarySchoolers());
+        $noTapSponsor->evaluations()->saveMany($this->veryfiesKids());
```
Comment (charlesstrange2) [inline database/seeds/SponsorsSeeder.php:48]: These last two adds rules to
- force primary school kids to be qualified by a younger kid
- force users to click a box to verify they have documentation

Not strictly necessary for this, we'll see if it testing spots it ;-)

**Review by charlesstrange2 — APPROVED** [review]
OK, give it  spin 🌪️

File: database/seeds/SponsorsSeeder.php (hunk 2 above)
Comment (AlinChindea) [reply database/seeds/SponsorsSeeder.php:48]: Should I remove these?

**Pull request merged**